lib/chart_helpers.dart

```diff
@@ -33,9 +33,19 @@ ChartOptions _generateBarChartOptions() {
     ..stacked = false
     ..scaleLabel = ScaleTitleOptions(labelString: labelString, display: true)
     ..ticks = (LinearTickOptions()..min = 0);
+
+  var tooltipLabelCallback = (ChartTooltipItem tooltipItem, ChartData data) {
+    var xLabel = data.datasets[tooltipItem.datasetIndex].label;
+    var yLabel = tooltipItem.yLabel;
+    var append = dataNormalisationEnabled ? '%' : '';
```

Comment:
s/append/suffix?

Reply:
Ah yes, fixed. Could only come up with 'appendix' as the noun form of append :/